### DISCUSSION TITLE
cannot add user

### DISCUSSION THREAD

**Discussion by susanforme** [verdaccio/verdaccio#4848] [Q&A] [answered]
# env
OS:windows 10;
# desc
When I pulled the image([verdaccio/verdaccio:latest](app://dd/dashboard/images/local/details?image=verdaccio%2Fverdaccio%3Alatest&imageId=sha256%3A960ec9d0fe83b18ed89b0bd9021d6affa745e661c431a4455ac41d8824eae5d8)) from Docker (without any other configuration), I couldn't add the user correctly; Stuck when adding users.

![image](https://github.com/user-attachments/assets/96184c99-9977-4eb8-9328-02c298ad64e2)

##  logs
this is my npm logs
```
0 verbose cli C:\Program Files\nodejs\node.exe C:\Program Files\nodejs\node_modules\npm\bin\npm-cli.js
1 info using npm@10.8.1
2 info using node@v20.16.0
3 silly config load:file:C:\Users\anonymous\AppData\Roaming\nvm\v20.16.0\node_modules\npm\npmrc
4 silly config load:file:C:\Users\anonymous\.npmrc
5 silly config load:file:C:\Users\anonymous\AppData\Roaming\npm\etc\npmrc
6 verbose title npm adduser
7 verbose argv "adduser" "--registry" "http://localhost:4873/"
8 verbose logfile logs-max:10 dir:C:\Users\anonymous\AppData\Local\npm-cache\_logs\2024-09-20T06_54_33_120Z-
9 verbose logfile C:\Users\anonymous\AppData\Local\npm-cache\_logs\2024-09-20T06_54_33_120Z-debug-0.log
10 notice Log in on http://localhost:4873/
11 verbose web adduser before first POST
12 silly logfile done cleaning log files
13 http fetch POST 404 http://localhost:4873/-/v1/login 41ms
14 verbose web add user not supported, trying couch

```

this is my docker logs
![image](https://github.com/user-attachments/assets/639e73bd-bb97-45a6-b756-f804a005f45e)
Can access the page normally
![image](https://github.com/user-attachments/assets/58b0c25e-82d3-4c38-9f01-b2306126c736)

**Comment by keinagae** [comment]
were you able to fix this issue  i am getting same issue

**Reply by keinagae** [reply]
@susanforme if you just enter your password here it work. For some reason it will show animated loading or some thing similar but you can just enter password

**Reply by juanpicado** [reply, marked as answer]
This has been already discussed, please search in the previous discussions
